client/settings/payment-request-section/index.js

```diff
@@ -1,178 +1,205 @@
+											appleLink: (
+												<a
+													target="_blank"
+													rel="noreferrer"
+													href="https://developer.apple.com/apple-pay/acceptable-use-guidelines-for-websites/"
+												/>
+											),
+											googleLink: (
+												<a
+													target="_blank"
+													rel="noreferrer"
+													href="https://androidpay.developers.google.com/terms/sellertos"
+												/>
+											),
+										},
+									} )
+									/* eslint-enable jsx-a11y/anchor-has-content */
 								}
-								onChange={ makeLocationChangeHandler( 'cart' ) }
-								label={ __(
-									'Cart',
+							</div>
+						</div>
+						<div className="express-checkout__link">
+							<a href={ customizeAppearanceURL }>
+								{ __(
+									'Customize',
 									'woocommerce-gateway-stripe'
 								) }
-							/>
+							</a>
+						</div>
+					</li>
+					{ displayLinkPaymentMethod && (
+						<li className="express-checkout has-icon-border">
+							<div className="express-checkout__checkbox loadable-checkbox label-hidden">
+								<CheckboxControl
+									label={ __(
+										'Link by Stripe Input',
+										'woocommerce-payments'
+									) }
+									checked={ isStripeLinkEnabled }
+									onChange={ updateStripeLinkCheckout }
+								/>
+							</div>
+							<div className="express-checkout__icon">
+								<LinkIcon size="medium" />
+							</div>
+							<div className="express-checkout__label-container">
+								<div className="express-checkout__label">
+									{ __(
+										'Link by Stripe',
+										'woocommerce-gateway-stripe'
+									) }
+								</div>
+								<div className="express-checkout__description">
+									{
+										/* eslint-disable jsx-a11y/anchor-has-content */
+										interpolateComponents( {
+											mixedString: __(
+												'Link autofills your customers’ payment and shipping details to ' +
+													'deliver an easy and seamless checkout experience. ' +
+													'New payment experience (UPE) needs to be enabled for Link. ' +
+													'By enabling this feature, you agree to the ' +
+													'{{stripeLinkTerms}}Link by Stripe terms{{/stripeLinkTerms}}, ' +
+													'and {{privacyPolicy}}Privacy Policy{{/privacyPolicy}}.',
+												'woocommerce-gateway-stripe'
+											),
+											components: {
+												stripeLinkTerms: (
+													<a
+														target="_blank"
+														rel="noreferrer"
+														href="https://link.co/terms"
+													/>
+												),
+												privacyPolicy: (
+													<a
+														target="_blank"
+														rel="noreferrer"
+														href="https://link.co/privacy"
+													/>
+												),
+											},
+										} )
+										/* eslint-enable jsx-a11y/anchor-has-content */
+									}
+								</div>
+							</div>
+							<div className="express-checkout__link">
+								{
+									/* eslint-disable jsx-a11y/anchor-has-content */
```

Comment:
🤏 These `eslint-disable` rules might not be necessary for this repository.

Reply:
Actually, tried removing it, and JS lint errors are thrown. We have to keep them.